CommandLine/libEGM/gmx.cpp

```diff
@@ -523,9 +521,9 @@ T* LoadResource(std::string fName, std::string type) {
   if (resType != type || resName.empty()) // trying to load wrong type (eg a.gmx has <b> instead of <a> as root xml)
     return nullptr;
 
-  int id = 0;
+  static std::unordered_map<std::string, int> ids;
```

Comment:
This being static actually doesn't matter right now. As far as I know, `LoadResource` here isn't actually being used yet except by the single resource loading functions below it which are also not being used. They would ideally be for importing a single GMX's resource into an existing project. It is expected that such a use case would provide an appropriate ID for the resource after loading it.

Reply:
shouldn't ids be a std::set?

Reply:
This is going to cause issues in server mode. LoadResource is being used. I know I implemented in background editor at least, I'm not sure about the rest

Reply:
I think you're right actually, the static is wrong. Regardless, libEGM really has no idea what ID to give a random GMX resource you asked it to load. That's the job of the caller to set a proper ID after calling it.

Reply:
Alright 50a548d0f31a548ff66479b47fc646ba278f48fa removed the static.